ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupViewDemoController.swift

```diff
@@ -189,7 +189,7 @@ class AvatarGroupViewDemoController: DemoController {
     private func insertAvatarViews(style: AvatarGroupViewStyle, showBorders: Bool) {
         var constraints: [NSLayoutConstraint] = []
 
-        for size in AvatarSize.allCases.reversed() {
+        for size in AvatarLegacySize.allCases.reversed() {
```

Comment:
Legacy would show an intent of being deprecated, do we want to discourage users from using existing Avatar view?

Reply:
Yes! Once we merge this to master, we do want to remove the Legacy one and encourage the client apps to update to the SwiftUI version of it (Vnext).